src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -641,6 +647,12 @@
             "title": "Source with echo by default",
             "description": "Whether to echo R code when sourcing it."
         },
+        "new_project_git_init": {
```

Comment:
This pref (and several others) were removed a while back; this change should not reintroduce them.

Reply:
Removed this one, and found two others (also removed)